system_baseline/mgmt_views/v0.py

```diff
@@ -21,7 +21,7 @@ def _update_baseline_counts():
     The baseline counts are updated when metrics are fetched via SQL
     """
     total_baselines = SystemBaseline.query.count()
-    total_accounts = SystemBaseline.query.distinct(SystemBaseline.account.name).count()
+    total_accounts = SystemBaseline.query.distinct(SystemBaseline.org_id.name).count()
```

Comment:
Do we need an if/else here also for those accounts without org_id yet?

Reply:
The same as above ^^

Reply:
Is the `.name` needed for distinct?
```
SystemBaseline.org_id.name
                     ^^^^^
```

Reply:
@vkrizan you are right, `#distinct` accepts `InstrumentedAttribute` too.